controllers/yawol-controller/loadbalancermachine/loadbalancermachine_controller.go

```diff
@@ -726,55 +804,76 @@ func (r *LoadBalancerMachineReconciler) deleteServer(
 func (r *LoadBalancerMachineReconciler) deletePort(
 	ctx context.Context,
 	osClient os.Client,
-	req ctrl.Request,
 	lbm *yawolv1beta1.LoadBalancerMachine,
-) error {
+) (bool, error) {
 	var err error
+	var requeue bool
 	var portClient os.PortClient
 	portClient, err = osClient.PortClient(ctx)
 	if err != nil {
-		return err
+		return false, err
 	}
 
-	if lbm.Status.PortID != nil {
-		if err = openstackhelper.DeletePort(ctx, portClient, *lbm.Status.PortID); err != nil {
+	if lbm.Status.DefaultPortID != nil {
+		if err = openstackhelper.DeletePort(ctx, portClient, *lbm.Status.DefaultPortID); err != nil {
 			switch err.(type) {
 			case gophercloud.ErrDefault404:
-				r.Log.Info("error deleting port, already deleted", "lbm", lbm.Name)
+				_ = helper.RemoveFromLBMStatus(ctx, r.Status(), lbm, "defaultPortIP")
```

Comment:
```suggestion
				_ = helper.RemoveFromLBMStatus(ctx, r.Client.Status(), lbm, "defaultPortIP")
```

Reply:
should be the same, i was just wondering why we use different clients here until i realised they are the same clients. maybe someone else is also questioning this in the future :D